README.md

```diff
@@ -175,6 +176,12 @@ A typical workflow starts with importing these classes from RocketPy:
 from rocketpy import Environment, Rocket, SolidMotor, Flight
 ```
 
+An optional step is to import datetime, which is used to define the date of the simulation:
```

Comment:
The diagram of classes needs to be refactored before v1, don't let me forget it please